pages/docs/sharing-data-externally/email-reports.mdx

```diff
@@ -0,0 +1,58 @@
+# E-mail Reports
+
+You can generate e-mail reports for any Saved Exploration or Dashboard in Glean.
+
+- [How to Send an E-mail Report](#how-to-send-an-e-mail-report)
+- [Setting Up E-mail Reports](#setting-up-e-mail-reports)
+- [Scheduling E-mail Reports](#scheduling-e-mail-reports)
+- [What Happens When Reports Fail](#what-happens-when-reports-fail)
+- [Managing Scheduled Reports](#managing-scheduled-reports)
+
+## How to Send an E-mail Report
+
+1. Go to the **Saved Exploration** or **Dashboard** you want to share.
+2. Click the **Share** button on the top right.
+3. Pick **Send as Email** from the dropdown.
+
+When sending an email report:
+
+- Specify a **Subject** of the report.
+- Add the **Email Addresses** you want to send to. You can e-mail a report to up to 25 recipients.
+- Write an optional **Message** if you want. This will be included in the body of the e-mail.
+
+## Options
+
+For **Saved Explorations**, you have a couple of extra options:
+
+1. **Include chart image**: This puts a picture of the chart in the e-mail.
+2. **Include underlying data**: This attaches the data as a `.csv` file. You can include up to 100,000 rows.
+
+For **Dashboards**, the e-mail always includes a full-length image of the dashboard.
+
+## Scheduling E-mail Reports
+
+You can configure recurring e-mail reports to be generated on a schedule by selecting "Schedule recurring report" at the bottom of the "Share as Email" form and then clicking "Set Schedule".
```

Comment:
Nit: I like the pattern you set above where we **bold** these references instead of "quoting" them.